test/unit/collections/CollectionsRepository.test.ts

```diff
@@ -0,0 +1,118 @@
+import { ReadError } from '../../../src'
+
+describe('CollectionsRepository', () => {
+  const sut: CollectionsRepository = new CollectionsRepository()
+  const testCollectionSuccessfulResponse = {
+    data: {
+      status: 'OK',
+      data: createCollectionPayload()
+    }
+  }
+  const testCollectionModel = createCollectionModel()
+
+  beforeEach(() => {
+    ApiConfig.init(
+      TestConstants.TEST_API_URL,
+      DataverseApiAuthMechanism.API_KEY,
+      TestConstants.TEST_DUMMY_API_KEY
+    )
+  })
+  describe('getCollection', () => {
+    const expectedRequestConfigApiKey = {
+      params: {
+        returnOwners: true
+      },
+      headers: TestConstants.TEST_EXPECTED_AUTHENTICATED_REQUEST_CONFIG_API_KEY.headers
+    }
+
+    describe('by numeric id', () => {
+      test('should return Collection when providing a numeric id', async () => {
+        jest.spyOn(axios, 'get').mockResolvedValue(testCollectionSuccessfulResponse)
+        const expectedApiEndpoint = `${TestConstants.TEST_API_URL}/dataverses/${testCollectionModel.id}`
+
+        // API Key auth
+        let actual = await sut.getCollection(testCollectionModel.id)
+
+        expect(axios.get).toHaveBeenCalledWith(expectedApiEndpoint, expectedRequestConfigApiKey)
+        expect(actual).toStrictEqual(testCollectionModel)
+
+        actual = await sut.getCollection(testCollectionModel.id)
+        expect(axios.get).toHaveBeenCalledWith(expectedApiEndpoint, expectedRequestConfigApiKey)
+        expect(actual).toStrictEqual(testCollectionModel)
+      })
+
+      test('should return error on repository read error', async () => {
+        jest.spyOn(axios, 'get').mockRejectedValue(TestConstants.TEST_ERROR_RESPONSE)
+        const expectedApiEndpoint = `${TestConstants.TEST_API_URL}/dataverses/${testCollectionModel.id}`
+        let error = undefined as unknown as ReadError
+
+        await sut.getCollection(testCollectionModel.id).catch((e) => (error = e))
+
+        expect(axios.get).toHaveBeenCalledWith(expectedApiEndpoint, expectedRequestConfigApiKey)
+        expect(error).toBeInstanceOf(Error)
+      })
+    })
+    describe('by alias id', () => {
+      test('should return a Collection when providing the Collection alias is successful', async () => {
+        jest.spyOn(axios, 'get').mockResolvedValue(testCollectionSuccessfulResponse)
+        const expectedApiEndpoint = `${TestConstants.TEST_API_URL}/dataverses/${testCollectionModel.alias}`
+
+        // API Key auth
+        let actual = await sut.getCollection(testCollectionModel.alias)
+
+        expect(axios.get).toHaveBeenCalledWith(expectedApiEndpoint, expectedRequestConfigApiKey)
+        expect(actual).toStrictEqual(testCollectionModel)
+
+        actual = await sut.getCollection(testCollectionModel.alias)
+        expect(axios.get).toHaveBeenCalledWith(expectedApiEndpoint, expectedRequestConfigApiKey)
+        expect(actual).toStrictEqual(testCollectionModel)
```

Comment:
Same here, the last 3 lines seem to be exactly the same as the first 3 lines
```suggestion
        let actual = await sut.getCollection(testCollectionModel.alias)

        expect(axios.get).toHaveBeenCalledWith(expectedApiEndpoint, expectedRequestConfigApiKey)
        expect(actual).toStrictEqual(testCollectionModel)
```